lib/jormungandr/test/integration/Cardano/Faucet.hs

```diff
@@ -1112,3 +1126,31 @@ genFaucets file n = do
             [ paymentAddress @'Testnet (publicKey $ addrXPrv ix)
             | ix <- [minBound..maxBound]
             ]
+
+sockAddrPort :: SockAddr -> Port a
```

Comment:
Why is this moved to the Faucet module?

Reply:
I moved `sockAddrPort` and `mkFeeEstimator` to Faucet as they are reused in `test/bench/Latency.hs` and `test/integration/Main.hs` and wanted to avoid duplication